packages/diagrams/frontend/sirius-components-diagrams/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@eclipse-sirius/sirius-components-diagrams",
+  "version": "2022.7.8",
+  "author": "Eclipse Sirius",
+  "license": "EPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-sirius/sirius-components"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "main": "./dist/sirius-components-diagrams.umd.js",
+  "module": "./dist/sirius-components-diagrams.es.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "require": "./dist/sirius-components-diagrams.umd.js",
+      "import": "./dist/sirius-components-diagrams.es.js"
+    }
+  },
+  "scripts": {
+    "build": "vite build && cp src/index.d.ts dist/ && mkdir -p dist/representation && cp src/representation/DiagramRepresentation.d.ts dist/representation/",
+    "format": "prettier --write \"src/**/*.{js,ts,tsx,css}\"",
+    "format-lint": "prettier --list-different \"src/**/*.{js,ts,tsx,css}\"",
+    "publish:local": "yalc push"
```

Comment:
Missing `test` script? Running `npx turbo run test` does not seem to execute diagram tests, and running `npm run test` from here obviously fails.

Reply:
That's on purpose, reflect metadata cannot work with vitest so I've removed the execution of the test for now.

You could add the test script and try that to see the error:

`npm run test --workspace=packages/diagrams/frontend/sirius-components-diagrams`